job_service/app.py

```diff
@@ -63,6 +64,17 @@ def handle_invalid_name(e):
     return {"message": str(e)}, 400
 
 
+@app.errorhandler(HTTPException)
+def handle_http_exception(e: HTTPException):
+    if str(e.code).startswith("4"):
+        logger.warning(e, exc_info=True)
+        error_message = str(e.description)
+    else:
+        logger.exception(e)
+        error_message = "Internal Server Error"
```

Comment:
HTTP error?

Reply:
Maybe OK to just keep str(e.description) in the else part as well? Or do we wish to simplify the message by writing HTTP error explicitly?

(I added internal server error based on the logic in your original suggestion for admin-backend)

Reply:
Yes, let's keep str(e.description) as job-service is our internal component. Admin-backend is different 👍